src/include/wiredtiger.in

```diff
@@ -3822,9 +3812,10 @@ struct __wt_file_system {
 	 * @param file_system the WT_FILE_SYSTEM
 	 * @param session the current WiredTiger session
 	 * @param name the name of the file system object
+	 * @param durable if the operation requires durability
 	 */
-	int (*fs_remove)(
-	    WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name);
+	int (*fs_remove)(WT_FILE_SYSTEM *file_system,
+	    WT_SESSION *session, const char *name, bool durable);
```

Comment:
> I prefer flags over a boolean. The API is only months old and we've found a reason to add a flag, which makes me think we'll probably want more in the future.

OK, pushed.

I used `WT_FS_DURABLE` as the `WT_FILE_SYSTEM.remove` and `WT_FILE_SYSTEM.rename` flag, my problem with using `WT_FS_OPEN_DURABLE` is not so much the flag range restriction, but the `OPEN` in the name.